rasa/shared/utils/validation.py

```diff
@@ -22,23 +24,103 @@
 KEY_TRAINING_DATA_FORMAT_VERSION = "version"
 
 
-class InvalidYamlFileError(ValueError):
+class InvalidYamlFileError(ValueError, RasaOpenSourceException):
```

Comment:
`InvalidYamlFileError` and `InvalidYamlFileException` have very similar name, I suggest we make one of them more specific

Reply:
should we rename this one to `InvalidYamlSchemaError`?

Reply:
:+1: renamed them to `YamlSyntaxException` and `YamlValidationException`

Reply:
great, that fixes it 👍